Tuturial-Structure-Template.md

```diff
@@ -1,68 +1,178 @@
+To successfully understand and complete this task you must have:
 
-When embedding code, be sure to include the language name to ensure the proper code highlighting when the markdown renders. For example, for javascript code snippets use: ```javascript  
+- A valid [Agora account](https://docs.agora.io/en/Agora%20Platform/sign_in_and_sign_up). If you don't have an account, see how to [Get started with Agoras](https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=PAGE_TITLE).
+  > The URL to the setup the Agora account is made up of _URL_:_PAGE_TITLE_. 
+  > For a blog named _How to Build a Token Server for Agora Applications using GoLang_ the link is: https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=How_to_Build_a_Token_Server_for_Agora_Applications_using_GoLang
+  > 
+  > It's imperative that you type this URL correctly as it will be the primary way Agora can properly track developer conversion from individual content posts.
+- An Agora project with the [app certificate](https://docs.agora.io/en/Agora%20Platform/manage_projects?platform=All%20Platforms#manage-your-app-certificates) enabled.  
+- Good knowledge of [Multi-factor authentication](https://en.wikipedia.org/wiki/Multi-factor_authentication).
 
-When embedding code using GitHub Gists, if the code is only a snippet make sure to denote that in the file name following the format: `functionName+snippet.fileExtension`  
 
-For example if my function name is generateToken and its a javascript snippet, the Gist would have the title: `generateToken+snippet.js`  
+## Project Setup 
 
-If the code within the Gist is an entire file or set of functions that can be copied and pasted and will work on their own as a library or project, you can just name the Gist after the file they would represent with the repository.  
+> Describe the steps to create a new project and set up the environment.
+> 
+> **IF NONE ARE NEEDED, REMOVE THIS SECTION**
+> 
+> For example:
 
-## Testing XYZ ## 
-> (use as section title; replace XYZ with the project or feature being tested)
+To create the environment necessary to **TITLE OF THIS PAGE**:
 
-Write the steps to test the project.  
+1. Do this.
+1. Do that.
+1. Do something else.
+
+> Markup commands to run in the terminal as ```code```.
+> 
+> If you need a specific file structure, do it as a text diagram:
+> For example: 
+> 
+> <environment_root> \
+> ├── index.html \
+> ├── scripts \
+> │ └── script.js \
+> └── styles \
+> │ └── style.css
+
+## Build \<A rewording of the page title>
+
+> If there are multiple logical sections in the procedure such as back-end and frond-end, use subsections. The goal is to modularize the various elements and build the project in a logical way that makes it easy for the user to follow along.
+> 
```

Comment:
Does an API call sequence diagram fit here?

Reply:
No, that will be in understand the tech.